core/base/libs/socrate/socrate/system.py

```diff
@@ -141,6 +141,7 @@ def clean_env():
         for item in os.environ.get('TLS', ALL_PORTS).split(','):
             if item in PORTS_REQUIRING_TLS:
                 os.environ[f'TLS_{item}']='True'
+    os.environ['CPU_COUNT'] = str(os.cpu_count())
```

Comment:
Do we want this overrideable from the outside?
```suggestion
    if 'CPU_COUNT' not in os.environ:
        os.environ['CPU_COUNT'] = str(os.cpu_count())
```